### DISCUSSION TITLE
❓ [Question] What is the difference between docker built on PyTorch NGC Container and PyTorch NGC Container?

### DISCUSSION THREAD

**Discussion by Guangyun-Xu** [pytorch/TensorRT#884] [Q&A]
## ❓ Question

Since PyTorch NGC 21.11+ already includes Torch-TensorRT, is it possible to use Torch-TensorRT directly in PyTorch NGC Container?

## What you have already tried

I read the README and tried to build docker according to it, but it keeps failing.

## Environment

> Build information about Torch-TensorRT can be found by turning on debug messages

 - PyTorch Version (e.g., 1.0):Do I need to install PyTorch locally?
 - CPU Architecture:AMD64/x64
 - OS (e.g., Linux):Linux
 - How you installed PyTorch (`conda`, `pip`, `libtorch`, source)：not installed
 - Build command you used (if compiling from source):
 - Are you using local sources or building from archives:
 - Python version:
 - CUDA version:
 - GPU models and configuration:
 - Any other relevant information:

**Comment by narendasan** [comment]
Yes you can use the precompiled torch-tensorrt inside the container
